api/slack/verification.go

```diff
@@ -1,15 +1,40 @@
 package slack
 
-import "net/http"
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// CheckMAC reports whether messageMAC is a valid HMAC tag for message.
+func CheckMAC(message, receivedMAC string, key []byte) bool {
```

Comment:
```suggestion
func checkMAC(message, receivedMAC string, key []byte) bool {
```

이건 private 으로 냅두는게 좋을 것 같습니다 아니면 테스트가 필요하구요 :smiley:

Reply:
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
> func CheckMAC(message, receivedMAC string, key []byte) bool {
> func checkMAC(message, receivedMAC string, key []byte) bool {
> Commit suggestion 
> Add suggestion to batch
> 이건 private 으로 냅두는게 좋을 것 같습니다 아니면 테스트가 필요하구요 😃

공부가 됩니다. 소문자 시작은 private 라는걸 배우고 이렇게 써먹네요. 
이런 부분이 golang 의 매력이다 싶습니다.